### ISSUE TITLE
4k is not at top of list when sorting videos by resolution column.

### ISSUE THREAD

**Issue by evanrich** [Tautulli/Tautulli#1056]
**Version:**  1.4.19

**Branch:** master

**Commit hash:** 	ea41d06023969d938bd3b690e72cd3de56b18370

**Operating system:** Linux 3.16.0-4-amd64

**Python version:** 2.7.13 (default, Apr 20 2017, 12:13:37) [GCC 6.3.0]

**What you did?**  sort video column by resolution

**What happened?** either get 720p  at the top, or 1080p...4k should be highest, but it shows in the middle (sorted by literal number)

**What you expected?**  720->1080-4k and vice verse.

You can reproduce this by having a 4k video in your library.  Click resolution column to sort each direction (ascending/descending).  4k should be at the top of the list but it isn't, it gets burried between 1080 and 720.

**Comment by evanrich** [comment]
this apperas to be in plexpy/data/interfaces/default/library.html, which calls via ajax
```
 // Populate watch time stats
        $.ajax({
            url: 'library_watch_time_stats',
            async: true,
            data: { section_id: section_id },
            complete: function(xhr, status) {
                $("#library-time-stats").html(xhr.responseText);
            }
        });
```

**Comment by JonnyWong16** [comment]
Because Python is sorting alphanumerically, instead of numerically. 

`1 < 4 < 7` or `7 > 4 > 1`

I will have to fix this later.

**Comment by evanrich** [comment]
agreed.   I had a tool that I wrote once and used bootstrap table to display data, the way I got around something like this was to store values as 720, 1080, 4096, and then use bootstrap table's format function to then conditionally format 4096 as 4k.   That's one thought.

**Comment by JonnyWong16** [comment]
4k is 2160p

**Comment by evanrich** [comment]
I'm aware.   but you aren't displaying 4k as "2k" or "2160p", so if your'e going to display it as "4k", then the method I proposed was to take the numerical representation of 4k as 4096.  It was a proposition.  if you're going to be formatting values upon display, then sure, you could store them as 2160 as well, and then convert the 2160 to 4k.. i was propsing one idea.

**Comment by JonnyWong16** [comment]
Fixed in v1.4.20.
